library/src/lib/time-picker/time-picker.component.ts

```diff
@@ -21,44 +21,64 @@ import { PopperOptions } from 'popper.js';
     styles: [':host {display: inline-block;}']
 })
 export class TimePickerComponent implements ControlValueAccessor, OnInit {
+
+    /** @Input An object that contains three integer properties: 'hour' (ranging from 0 to 23),
+     * 'minute' (ranging from 0 to 59), and 'second' (ranging from 0 to 59).  */
     @Input()
     time: TimeObject = { hour: 0, minute: 0, second: 0 };
 
+    /** @Input Uses compact time picker. */
     @Input()
     compact: boolean = false;
 
+    /** @Input Boolean, when set to true, uses the 24 hour clock (hours ranging from 0 to 23)
+     * and does not display a period control. Default is false. */
     @Input()
     meridian: boolean;
 
+    /** @Input Disables the component. */
     @Input()
     disabled: boolean;
 
+    /** @Input Boolean, when set to false, hides the buttons that increment and decrement the corresponding input. Default is true. */
     @Input()
     spinners: boolean = true;
 
+    /** @Input Boolean, when set to false, hides the input for seconds. Default is true. */
     @Input()
     displaySeconds: boolean = true;
 
+    /** @hidden */
     @ViewChild(TimeComponent)
     child: TimeComponent;
 
+    /** @hidden */
     period: string;
 
+    /** @hidden */
     isOpen: boolean;
 
+    /** @hidden */
     placeholder: string;
 
+    /** @hidden */
     onChange: Function = () => {};
+    /** @hidden */
     onTouched: Function = () => {};
 
+    /** @hidden */
     ngOnInit(): void {
         this.placeholder = this.getPlaceholder();
     }
 
+    /**
+     * @returns Returns the current value of the time input.
+     */
     getTime() {
```

Comment:
If this returns something, the return type should be explicitly given by the function.

In that case, you would not need the **@returns**, typedoc will infer the return type. You'd just need a small explanation of what the method does.